### PR TITLE
garbage collection: add blurb about ImageMaximumGCAge

### PR DESCRIPTION
**Pull request by haircommander**

### CODE REVIEW

**Comment by katcosgrove** [comment]
Hi, @haircommander! v1.29 Docs Lead here. Please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday 14th November 2023. Thank you!

**Comment by katcosgrove** [comment]
Hi, @haircommander! Just dropping by to remind you that the deadline for docs PRs to be ready for review is tomorrow, November 14. Is there anything we can help you with to get this PR ready?

**Comment by haircommander** [comment]
thanks for the reminder! I think a small addition is all that is needed for now. I've updated the document for GC to include a note about the existence of the new feature. As it matures, we can evaluate adding a full blog.

**Comment by katcosgrove** [comment]
Hi, @haircommander! Kubernetes v1.29 Docs Lead here. Just a reminder that the deadline to have this PR reviewed and merged is Tuesday, 28 November. Let me know if you need any help!

File: content/en/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -137,6 +137,13 @@ collection, which deletes images in order based on the last time they were used,
 starting with the oldest first. The kubelet deletes images
 until disk usage reaches the `LowThresholdPercent` value.
 
+As of v1.29.0, the kubelet supports a new configuration field:
+
+* `ImageMaximumGCAge`
+
+This field, when enabled with the alpha feature gate of the same name, will set the
+maximum time an image can be unused for, regardless of disk usage.
+
```
Comment (sftim) [inline content/en/docs/concepts/architecture/garbage-collection.md:146]: Please move this into a new section that comes right at the end of the `## Configuring garbage collection` section. 

I suggest, based on our style guide and conventions:
```diff
-As of v1.29.0, the kubelet supports a new configuration field:
-
-* `ImageMaximumGCAge`
-
-This field, when enabled with the alpha feature gate of the same name, will set the
-maximum time an image can be unused for, regardless of disk usage.
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+As an alpha feature, you can specify the maximum time a local image can be unused for,
+regardless of disk usage. This is a kubelet setting that you configure for each node.
+To configure the setting, enable the `ImageMaximumGCAge`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the kubelet,
+and also set a value for the `ImageMaximumGCAge` field in the kubelet configuration file.
+The value is specified as a Kubernetes _duration_; for example, you can set the configuration
+field to `3d12h`, which means 3 days and 12 hours.
```

**Comment by katcosgrove** [comment]
Hello @haircommander! The deadline to have this PR merged is **Tuesday, November 28**. Enhancements that do not include required documentation may not be included in the release. Is there anything I can do to help get this ready to merge? Thanks!

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**